MekHQ/build.gradle

```diff
@@ -554,7 +553,7 @@ task javadocJar(type: Jar) {
 publishing {
     publications {
         publishMMLibrary(MavenPublication) {
-            artifactId = "mekhq${mmBranchTag}"
+            artifactId = "mekhq${hqBranchTag.replaceAll('[^A-Za-z0-9_\\-.]','-')}"
```

Comment:
Some characters (notably `/`) are allowed for branch names but not for maven artifact names (I changed this in all three projects - no need to check).